usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+def test_sles_hardened(host, get_release_value, is_sles_sap, swap_file):
+    variant = get_release_value('VARIANT_ID') or ''
+
+    if "hardened" not in variant:
+        pytest.skip('Skipping non-hardened image')
+
+    xml_path = "pub/projects/security/oval/suse.linux.enterprise.15.xml"
+    # Downloaded file should have slashes replaced by hyphens
+    xml_file = xml_path.replace("/", "-")
+    oscap_dir = "/tmp/oscap"
+    oscap_file = "/usr/share/xml/scap/ssg/content/ssg-sle15-ds.xml"
+    oscap_profile = "pcs-hardening-sap" if is_sles_sap() else "pcs-hardening-sap"
```

Comment:
Typo in the else value. I assume that would be s/pcs-hardening-sap/pcs-hardening?

Reply:
Fixed. Thanks!